R/pkg/R/functions.R

```diff
@@ -907,6 +907,30 @@ setMethod("initcap",
             column(jc)
           })
 
+#' @details
+#' \code{isinf}: Returns true if the column is Infinity.
+#' @rdname column_nonaggregate_functions
+#' @aliases isnan isnan,Column-method
+#' @note isinf since 2.4.0
+setMethod("isinf",
```

Comment:
We don't need to have duplicate method definition like this. Maybe we can follow `isNaN`'s approach if we really need to have both `isinf` and `isInf`.

Reply:
Would it be alright if I omit `isInf` completely and only implement `isinf`?

Reply:
For `isNaN` case, `functions.R` only defines `isnan`. `isNaN` is defined in `column.R`. If we really want to have both `isInf` and `isinf` like `isNaN` and `isnan`, maybe to follow it instead having two duplicate method definition here is better.